modules/perception/obstacle/onboard/fusion_subnode.cc

```diff
@@ -82,11 +82,23 @@ bool FusionSubnode::InitInternal() {
 
   lane_objects_.reset(new LaneObjects());
 
+  // // CIPV data
+  // cipv_object_data_ = dynamic_cast<CIPVObjectData *>(
+  //     shared_data_manager_->GetSharedData("CIPVObjectData"));
```

Comment:
looks like cipv_object_data_ is not used in fusion subnode yet?

Reply:
Yes. Currently CIPV_object_data is not used as a output yet. We will discuss and decide either remove or keep CIPVObjectData later.